Microsoft.Toolkit.Uwp.UI.Controls/HamburgerMenu/HamburgerMenu.Properties.cs

```diff
@@ -111,6 +116,15 @@ public double CompactPaneLength
         /// <summary>
         /// Gets or sets the Brush to apply to the background of the Pane area of the control.
         /// </summary>
+        public Brush AccentColor
```

Comment:
News, Money, Sports each have an AccentColor (varies), PaneColor (#2B2B2B) and a Top Nav Color (#F2F2F2).